protopipe/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.2.1-dev"
```

Comment:
Usually "0.y.z" already states "dev" no?

Reply:
I am not an expert on this: this "dev" means that is the development version after the tagging of the last release (the docs will be built from the master branch as for ctapipe, which code to get the version is a bit complicated for me now.).
I have seen people using "-alpha", "-beta", but I didn't want to be overly specific for the moment.

Reply:
I think in this case, "dev" means "not a tagged release"

Reply:
> I think in this case, "dev" means "not a tagged release"

Yes, this is a better interpretation...can we keep it like this or do you suggest another naming?